src/components/common/input/PasswordInput.tsx

```diff
@@ -14,6 +14,7 @@ const PasswordInput = forwardRef<HTMLInputElement, PasswordInputProps>(
     return (
       <div className="relative w-full">
         <Input
+          autoComplete="current-password"
```

Comment:
이 자동완성 설정을 공용 컴포넌트에서 하드코딩해도 되나요?
그럼 `PasswordInput`을 사용하는 컴포넌트에서 다른 `autoComplete` 값을 지정하면, 그 값이 기본값을 오버라이드하게 되지 않을까요..? 현재 회원가입/로그인/비밀번호 재설정 모두 이 컴포넌트를 사용 중이라 문제가 되지 않을까 싶습니다. 
여러 input에 `autoComplete`를 추가하신 것으로 보이는데, 이 속성을 `props`로 받는 방식은 어떨까요?

```suggestion
          autoComplete={autoComplete}
```

Reply:
PasswordInput은 이름에서부터 비밀번호를 입력받을 때 사용하는 input이라 생각해서 여기만 직접 하드코딩? 했습니다
PasswordInput이 비밀번호 입력이 아닌 경우에도 사용이 될까요??

Reply:
로그인, 비밀번호 재설정, 회원가입 모두 비밀번호를 입력받는데, 회원가입/비밀번호 재설정에서는 "current-password"이면 안되는거 아닌가요? 이 두 경우는 autoComplete="new-password"가 되어야 하는데 PasswordInput에서 "current-password"로 정해두면 모달에서 변경이 안되지 않나요?

Reply:
아~ new-password 옵션.. 제가 약간 흐린 눈 하고있었네요
수정하겠습니다!

Reply:
✅ f1aa2b3ea7f059ba8dbab821d3f15ccbe5d57510

Reply:
prop으로 안받고 그냥 개별 지정했습니다!